qemu/tests/cfg/block_hotplug_check.cfg

```diff
@@ -2,6 +2,7 @@
     type = pci_hotplug_check
     only Host_RHEL
     only virtio_net virtio_blk
+    no spapr_vscsi
```

Comment:
Since `only virtio_net virtio_blk` has been added above, there is no need to modify this cfg file.